ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/driver/AssertExecutor.java

```diff
@@ -60,41 +61,50 @@
   static final List<SourceProperty> MUST_MATCH = ImmutableList.<SourceProperty>builder()
       .add(new SourceProperty(
           DataSource::getSchema,
-          cs -> cs.getElements().toLogicalSchema(),
+          (cs, cfg) -> cs.getElements().toLogicalSchema(),
```

Comment:
Added tests to ensure that YATT is properly verifying `KEY_FORMAT` of sources, and that `FORMAT` behaves as expected too. Didn't add tests to check that default format values are picked up from the config since there are already QTT tests for that, and I don't see the value of duplication here.